cpp/src/io/parquet/experimental/page_index_filter.cu

```diff
@@ -668,6 +485,189 @@ struct page_stats_caster : public stats_caster_base {
   }
 };
 
+/**
+ * @brief Functor to build the NEXT Fenwick tree level from the current level data
+ *
+ * @param tree_level_ptrs Pointers to the start of Fenwick tree level data
+ * @param current_level Current tree level
+ * @param current_level_size Size of the current tree level
+ * @param next_level_size Size of the next tree level
+ */
+struct build_next_fenwick_tree_level_functor {
```

Comment:
this could also be called `build_fenwick_tree_level_functor`, with parameters
`tree_level_ptrs`, `prev_level`, `prev_level_size`, `level_size`.
So we don't need to emphasize NEXT in the name and docs.